SimPEG/Directives.py

```diff
@@ -144,6 +144,35 @@ def endIter(self):
             if self.debug: print 'BetaSchedule is cooling Beta. Iteration: %d' % self.opt.iter
             self.invProb.beta /= self.coolingFactor
 
+#class BetaSchedule_PGN_CG(InversionDirective):
```

Comment:
Yea this was my first kick at the cat.
I commented it out so I can steal from it. If you still have strong feelings for a directive, we can go back to it. Otherwise, I would just kill it.